dataset/README.md

```diff
@@ -0,0 +1,59 @@
+# Input data
+The sampling and inference steps requires a csv referencing input data. An example of input csv can be found in [tests](tests/sampling/sampling_segmentation_binary_ci.csv).
+Each row of this csv is considered, in geo-deep-learning terms, to be an [AOI](https://torchgeo.readthedocs.io/en/latest/user/glossary.html#term-area-of-interest-AOI).
+
+| raster path               | vector ground truth path | dataset split | aoi id (optional) |
+|---------------------------|--------------------------|---------------|-------------------|
+| my_dir/my_geoimagery1.tif | my_dir/my_geogt1.gpkg    | trn           | Ontario-1         |
+| my_dir/my_geoimagery2.tif | my_dir/my_geogt2.gpkg    | tst           | NewBrunswick-23   |
+| ...                       | ...                      | ...           | ...               |
+
+> The use of aoi id information will be implemented in a near future. It will serve, for example, to print a detailed report of sampling, training and evaluation, or for easier debugging.
+
+The path to a custom csv must be entered in the [dataset configuration](https://github.com/NRCan/geo-deep-learning/blob/develop/config/dataset/test_ci_segmentation_binary.yaml#L9). See the [configuration documentation](config/README.md) for more information.
+Also check the [suggested folder structure](https://github.com/NRCan/geo-deep-learning#folder-structure).
+
+## Dataset splits
+Split in csv should be either "trn", "tst" or "inference". The validation split is automatically created during sampling. It's proportion is set by the [dataset config](https://github.com/NRCan/geo-deep-learning/blob/develop/config/dataset/test_ci_segmentation_binary.yaml#L8). 
```

Comment:
... automatically created from the "trn" split, during sampling.

Reply:
Thanks, I'll correct this in a next PR.